src/pages/Chaehyun/InputList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Input from './Input';
+import './InputList.scss';
+
+function InputList({ signup }) {
+  return (
+    <div className="signUpBox">
+      {signup.map(data => (
```

Comment:
data라는 범용적인 변수명보다는 좀 더 구체적으로 변수명을 지어주세요

Reply:
```suggestion
      {signup.map(({type, text, id, placholder}) => (
```
매개변수자리에서 바로 구조분해할당 가능합니다!